3723/3/app/models/place.rb

```diff
@@ -0,0 +1,9 @@
+class Place < ActiveRecord::Base
+  validates :name, :location, :description, :rating, presence: true
+
+  has_many :comments
+
+  def self.update_rating(place)
```

Comment:
First of all, you have created a method which called by `Place` class and provides to its object of the same class, instead of calling method on the object of the class

Reply:
Secondly, I meant that `comments.average(:rating)` this code will appear within `Place` model